src/test/java/uk/gov/justice/digital/hmpps/vsip/pages/NeedAReviewPage.java

```diff
@@ -14,21 +14,16 @@ public void isSummaryListOptionDisplayed(String summaryList) {
         methodsService.isElementDisplayed("xpath", "//span[contains(@class, 'govuk-details__summary-text')]");
     }
 
-    public void clickOnView()
-    {
+    public void clickOnView(String excludedate) {
+        String xpath = "//li[@class='govuk-!-margin-bottom-0' and contains(text(), '" + excludedate + "')]";
```

Comment:
I think we need a different selector for this. `govuk-!-margin-bottom-0` is just a styling class that we use all over the place so can't be relied upon.

Reply:
Done ~ changed the xpath